lib/rex/proto/kerberos/credential_cache/krb5_ccache_credential_address.rb

```diff
@@ -0,0 +1,36 @@
+# -*- coding: binary -*-
+
+require 'bindata'
+
+require 'rex/proto/kerberos/credential_cache/primitive'
+
+module Rex::Proto::Kerberos::CredentialCache
+  class Krb5CcacheCredentialAddress < BinData::Record
+    ADDRTYPE_INET  = 0x0002 # https://web.mit.edu/kerberos/krb5-devel/doc/appdev/refs/macros/ADDRTYPE_INET.html
+    ADDRTYPE_INET6 = 0x0018 # https://web.mit.edu/kerberos/krb5-devel/doc/appdev/refs/macros/ADDRTYPE_INET6.html
```

Comment:
Not a blocker; We've these constants over here too https://github.com/adfoster-r7/metasploit-framework/blob/3f56f9891d0d80a01061f975e4b1019b57f800fb/lib/rex/proto/kerberos/model/address_type.rb